src/test/java/uk/gov/dvsa/mot/fixtures/mot/PurchaseHistoryPageStepDefinitions.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.dvsa.mot.fixtures.mot;
+
+import static junit.framework.TestCase.assertTrue;
+
+import cucumber.api.java8.En;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.dvsa.mot.framework.WebDriverWrapper;
+
+import javax.inject.Inject;
+
+/**
+ * Step definitions specific to the purchase history page of the mot app.
+ */
+public class PurchaseHistoryPageStepDefinitions implements En {
```

Comment:
Can we rename this to be CPMSStepDefinitions so it is more generic and can be used for other CPMS scenarios rather than specific to the one scenario.